socorro/external/es/index_cleaner.py

```diff
@@ -37,11 +37,17 @@ def __init__(self, config):
         super(IndexCleaner, self).__init__()
         self.config = config
 
-    def delete_old_indices(self):
```

Comment:
This PR ended up taking longer than I expected because the original plan of using this method with a `retention_policy` of 0 meant that indices for future weeks (such as those created by the scripts in this PR) would not actually be deleted. I figured refactoring this class while retaining it's existing API was the cleanest fix.